### PR TITLE
 tables good, empty tables gooood

### PR DESCRIPTION
**Pull request by rosasck**

### CODE REVIEW

**Comment by BroughtMyPopcorn** [comment]
Could be wrong, but I think we wanted to have the table headers shown even if there wasn't anything to display, now there's just nothing showing on these pages.

**Review by ericharper1 — CHANGES_REQUESTED** [review]
This occurred when I tried to delete a student response from my researcher table. :(

It gets triggered at line 50 in researcher_home.py. Seems to be that it doesn't like rendering the context variable in the template when there is nothing to display.

image

File: simcon_project/simcon_project/settings.py [hunk 1]
```diff
@@ -137,7 +137,7 @@
 EMAIL_HOST = 'smtp.gmail.com'
 EMAIL_PORT = 587
 EMAIL_HOST_USER = 'simcon.dev@gmail.com'
-EMAIL_HOST_PASSWORD = 'zhtsjuriiabkdahg'
+EMAIL_HOST_PASSWORD = 'FAKEPASSWORD1102302032198'
```
Comment (Likhovodov) [inline simcon_project/simcon_project/settings.py]: Undo

**Review by Likhovodov — APPROVED** [review]
Looks good. If we could get a row indicating that the table is empty somehow, it would be great though it's far from a necessity.

**Review by JerCarnahan — APPROVED** [review]

**Pull request merged**